sjb/config/common/test_cases/origin_release_install_gce.yml

```diff
@@ -121,7 +121,7 @@ extensions:
           docker pull openshift/origin-gce:latest
         fi
 
-        ../../bin/local.sh ansible-playbook -e "provision_gce_docker_storage_driver=${DOCKER_STORAGE_DRIVER}" -e "openshift_test_repo=${location_url}" playbooks/launch.yaml
+        ../../bin/local.sh ansible-playbook -e "provision_gce_docker_storage_driver=${DOCKER_STORAGE_DRIVER}" -e "openshift_test_repo=${location_url}" -e "openshift_enable_service_catalog=true" -e "template_service_broker_install=true" playbooks/launch.yaml
```

Comment:
@smarterclayton PTAL

Reply:
Wrong place, add it to openshift/release cluster/test-deploy/data/vars.yaml